### ISSUE TITLE
FN6 Tweaks

### ISSUE THREAD

**Issue by siwelwerd** [TeamBasedInquiryLearning/precalculus#183]
- [Activity 2.6.2](https://teambasedinquirylearning.github.io/precalculus/2024/instructor/FN6.html#activity-temp-conversion) I think we are dating ourselves with references to nightly TV news... maybe rephrase in terms of the weather app on your phone?
- I guess it would detract from the lesson, but I always think it's fun to have them derive this conversion from a the phase change points of water (i.e. (0,32) and (100,212)).
- [Activity 2.6.14(a)](https://teambasedinquirylearning.github.io/precalculus/2024/instructor/FN6.html#activity-find-inverse-xy-2) 
I'm really not a fan of "interchanging the variables"... I much prefer a sense-making approach of solving for $x$ in terms of $y$.

**Comment by kathypinzon** [comment]
I mentioned to @AbbyANoble that she could have them come up with this temperature model in LF5.  So maybe we could refer to it there.

**Comment by siwelwerd** [comment]
Closed by #205
